packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -205,15 +216,26 @@ export default function CodeTextArea({
   onCtrlEnter,
   wrapperClassName,
   completions,
+  resizable = false,
+  defaultHeight = 200, // for resizable
```

Comment:
question: Does this apply even if resizable is `false`? I would expect so, so maybe we need to remove the comment?

Reply:
unfortunately no, when not resizable and when applying the defaultHeight, the button layout gets messed up.

Can look more into some of these weird layout issues later, or clean up the combinations that will never be used. But it doesn't break for any of our current usages.